cli.ts

```diff
@@ -11,6 +11,7 @@ import { getSeam } from "./lib/get-seam"
 import chalk from "chalk"
 import { interactForServerSelection } from "./lib/interact-for-server-selection"
 import { getServer } from "./lib/get-server"
+import { getApiDefinitions } from "./lib/get-api-definitions"
```

Comment:
Seam projects use no semis and quotes, i think this got formatted, we should drop in a prettierrc

Reply:
Yep, actually added prettier, but I'm hand-reverting the changes here to add it in a separate PR